### ISSUE TITLE
Command Palette: No longer closes on Escape key

### ISSUE THREAD

**Issue by mirka** [WordPress/gutenberg#59080]
The Command Palette used to close on <kbd>Esc</kbd>, as can be observed in the [WP Playground for WP 6.4](https://playground.wordpress.net/?php=8.0&wp=6.4&storage=none). This is no longer the case in `trunk`.

## Steps to reproduce

1. In the post editor, invoke the Command Palette (<kbd>⌘K</kbd>).
2. Hit <kbd>Esc</kbd>.

### Expected behavior

The Command Palette closes.

### Actual behavior

The Command Palette doesn't close.

**Comment by t-hamano** [comment]
As the screenshot below shows, I was unable to reproduce it in my environment. IME input also seems to be handled properly.

https://github.com/WordPress/gutenberg/assets/54422211/a886598b-0f22-4d01-b9b4-1674c04929af

Which OS/browser is this issue occurring on? I tested in the following environment.

- OS: Windows OS
- WordPress Version: WordPress 6.5-beta1-57631
- Gutenberg Version: 17.7.0 (trunk)
- Browser: Chrome 121.0.6167.162, Firefox 122.0.1

**Comment by mirka** [comment]
I did some more testing. With WP Playground 6.5-beta1 on a Mac, the bug reproduces in Chrome but not in Safari/Firefox. However, I tried again with an Incognito tab and also on a different Mac (with different Chrome settings), and it did not reproduce.

It's just my specific Chrome setup that somehow exhibits this difference between WP 6.4 and 6.5 beta 😅 I have no idea what it is, but I'm going to have to close this as something on my end.

Thanks for testing!

**Comment by mirka** [comment]
Noting for anyone who sees this same issue — the culprit for me was the [Vimium extension](https://chromewebstore.google.com/detail/vimium/dbepggeogbaibhgnhhndojpepiihcmeb) for Chrome. You will need to add disable URL rules for anywhere you use wp-admin on.
